src/views/components/common/trace-detail-statistics-table.vue

```diff
@@ -0,0 +1,314 @@
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+<template>
+  <div class="trace-detail-chart-table">
+    <div class="rk-trace-t-loading" v-show="loading">
+      <svg class="icon loading">
+        <use xlink:href="#spinner"></use>
+      </svg>
+    </div>
+    <TraceContainer :tableData="tableData" :type="HeaderType">
+      <div class="trace-tips" v-if="!tableData.length">{{ $t('noData') }}</div>
+    </TraceContainer>
+  </div>
+</template>
+<style lang="scss">
+  .rk-tooltip-popper.trace-table-tooltip .rk-tooltip-inner {
+    max-width: 600px;
+  }
+  .trace-detail-chart-table {
+    position: relative;
+    min-height: 300px;
+    border-bottom: 1px solid #ccc;
+  }
+</style>
+
+<script lang="js">
+  import copy from '@/utils/copy';
+  import TraceContainer from './trace-chart-table/trace-statistics-container';
+  import _ from 'lodash';
+  /* eslint-disable */
+  /* tslint:disable */
+  export default {
+    components: {
+      TraceContainer,
+    },
+    props: ['data', 'traceId', 'showBtnDetail', 'HeaderType'],
+    watch: {
+      data(val, oldVal) {
+        if (!this.data.length) {
+          this.tableData = [];
+          return;
+        }
+        this.tableData = this.compute(this.changeTree());
+        this.loading = false;
+      },
+    },
+    data() {
+      return {
+        tableData: [],
+        diaplay: true,
+        list: [],
+        loading: true,
+      };
+    },
+    methods: {
+      copy,
+      compute(data){
```

Comment:
Could you add a `if` to `return`, when the component isn't the statistics table. And move the function to `trace-detail-chart-table.vue`. Other functions seem to look the same. WDYT @wu-sheng @Jtrust  

Should the statistics table be same as the other three types to support clicking to view trace span logs? @wu-sheng

Reply:
@Fine0830 What do you want me to confirm?

Reply:
uh....I mean, what do you think of this suggestion? And is this function supported?
![2](https://user-images.githubusercontent.com/20871783/118106635-60b6c800-b410-11eb-83bc-82f1f002d1c7.png)

Reply:
Why do you move the statistic page into the span view? This PR is making a statistics for all spans, right? It tries to find the slowest or hot spans, from my understanding.
I am not 100% sure.

Reply:
Yeap. It doesn't seem necessary

Reply:
This PR merges the span according to operation name and get statistics data calculated in the browser. So, you need to recheck the algorithm. @Fine0830